docs/admin_api/media_admin_api.md

```diff
@@ -22,19 +22,55 @@ It returns a JSON body like the following:
 }
 ```
 
-# Quarantine media in a room
+# Quarantine media
 
-This API 'quarantines' all the media in a room.
+Quarantining media means that it is marked as inaccessible by users. It applies
+to any local media, and any locally-cached copies of remote media.
 
-The API is:
+The media file itself (and any thumbnails) is not deleted from the server.
+
+## Quarantining media by ID
+
+This API quarantines a single piece of local or remote media.
 
 ```
-POST /_synapse/admin/v1/quarantine_media/<room_id>
+POST /_synapse/admin/v1/quarantine_media_by_id/<server_name>/<media_id>
 
 {}
 ```
 
-Quarantining media means that it is marked as inaccessible by users. It applies
-to any local media, and any locally-cached copies of remote media.
+Where `server_name` is in the form of `example.org`, and `media_id` is in the
+form of `abcdefg12345...`.
 
-The media file itself (and any thumbnails) is not deleted from the server.
+## Quarantining media in a room
+
+This API quarantines all local and remote media in a room.
+
+```
+POST /_synapse/admin/v1/quarantine_media_by_room/<room_id>
+
+{
+  "num_quarantined": 10  # The number of media items successfully quarantined
```

Comment:
is this a thing you need to pass in the POST body? the doc makes it look that way. likewise below

Reply:
Very good point.